dash/dash.py

```diff
@@ -276,10 +302,21 @@ def serve_layout(self):
         )
 
     def _config(self):
-        return {
+        config = {
             'url_base_pathname': self.url_base_pathname,
             'requests_pathname_prefix': self.config['requests_pathname_prefix']
         }
+        if self._reload_hash:
+            config['hot_reload'] = {
+                'interval': self._reload_interval
+            }
+        return config
+
+    def serve_reload_hash(self):
+        return flask.jsonify({
+            'reloadHash': self._reload_hash,
+            'hard': self._hard_reload
```

Comment:
It doesn't look like the 'hard' variable is used at all in the front-end. Can we just remove it here, in `_on_assets_change` and in `__init__`?

Reply:
It is used to trigger a soft reload.
https://github.com/plotly/dash-renderer/blob/71a42e2f63880ec8ce2cbc01a8e18e582857f97d/src/components/core/Reloader.react.js#L35

Reply:
Oh okay, I see. So is this variable just set to true on the first asset change? I see it gets set to True in `_on_assets_change` but don't see it getting set back anywhere

Reply:
Good catch, need to reset it after serving the response.